src/main/java/net/smartcosmos/dto/metadata/MetadataCreate.java

```diff
@@ -0,0 +1,37 @@
+package net.smartcosmos.dto.metadata;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Setter;
+
+import java.beans.ConstructorProperties;
+import java.util.HashMap;
+import java.util.Map;
+
+@Data
+@JsonIgnoreProperties(value = {"version"}, ignoreUnknown = true)
+public class MetadataCreate {
+    private static final int VERSION = 1;
```

Comment:
Why should the value be in the name of the constant? I can't imagine any situation where we would have multiple version constants in a model.

The intention behind it was to ensure the same value is used for the field and the constructor for the builder.